zone/spell_effects.cpp

```diff
@@ -7018,79 +7018,83 @@ bool Mob::TryDeathSave() {
 		int64 HealAmt = 300; //Death Pact max Heal
 
 		if(buffSlot >= 0){
-
 			UD_HealMod = buffs[buffSlot].ExtraDIChance;
+			SuccessChance = ((GetCHA() * (RuleI(Spells, DeathSaveCharismaMod))) + 1) / 10; //(CHA Mod Default = 3)
 
-			SuccessChance = ( (GetCHA() * (RuleI(Spells, DeathSaveCharismaMod))) + 1) / 10; //(CHA Mod Default = 3)
-
-			if (SuccessChance > 95)
+			if (SuccessChance > 95) {
 				SuccessChance = 95;
+			}
 
 			if(zone->random.Roll(SuccessChance)) {
-
-				if(spellbonuses.DeathSave[SBIndex::DEATH_SAVE_TYPE] == 2)
+				if(spellbonuses.DeathSave[SBIndex::DEATH_SAVE_TYPE] == 2) {
```

Comment:
`if (`

Reply:
Could we make a namespace similar to `Race`/`Class` for this? What does `2` represent here? Trying to avoid magic numbers if we can.

Reply:
1 = 300 HP Healed, 2 = 8000 HP Healed

Not sure how that would get setup. Might seem overly verbose.

Reply:
```cpp
namespace DeathSave {
	constexpr uint32 HP300  = 1;
	constexpr uint32 HP8000 = 2;
}
```

Reply:
Thank you